Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -52,6 +53,17 @@ void __InitDWriteFontPropertiesFromName(
     return s_dwriteFactory;
 }
 
+// Private helper for converting a DWRITE_FONT_METRIC unit to a CTFont API unit
+// DWRITE_FONT_METRICS measures its metrics in 'design units'
+// CTFont APIs generally return in terms of point size
+CGFloat _CoreTextScaleMetric(CGFloat metric, CGFloat pointSizeUnitsPerEm) {
+    // design units * (pt / em) / (design units / em)
+    // = design units * (pt / em) * (em / design units)
+    // = pt * (design units / design units) * (em / em)
+    // = pt
+    return metric * pointSizeUnitsPerEm;
```

Comment:
>return metric * pointSizeUnitsPerEm; [](start = 4, length = 36)

uhh, this just evaluates to

design unit * pt/em

which doesn't reduce to anything... what final unit were you hoping to get out here?